frappe/public/js/frappe/views/reports/report_view.js

```diff
@@ -641,12 +641,21 @@ frappe.views.ReportView = class ReportView extends frappe.views.ListView {
 	}
 
 	set_fields() {
+		// remove fields that don't exist any more
+		let remove_non_existent_fields = fields =>
+			fields.filter(f =>
+				frappe.model.std_fields_list.includes(f[0])
+				|| frappe.meta.has_field(f[1], f[0])
```

Comment:
This case is already handled by the `_add_field` method. Can you use that instead?